app/assets/src/components/common/CoverageVizSidebar/coverage_viz_sidebar.scss

```diff
@@ -0,0 +1,136 @@
+@import "~styles/themes/colors";
+@import "~styles/themes/typography";
+
+.contents {
+  max-width: 1280px;
+  margin: 0 auto;
+  margin-top: 30px;
+  margin-bottom: 40px;
+  width: 96%;
+  text-align: left;
+}
+
+.header {
+  margin-bottom: 30px;
+  display: flex;
+
+  .headerText {
+    .title {
+      @include font-header-l;
+    }
+
+    .subtitle {
+      @include font-body-xs;
+    }
+  }
+
+  .fill {
+    flex: 1 1 0;
+    min-width: 0;
+  }
+
+  .headerControls {
+    .vizLinkContainer {
+      text-align: right;
+      margin-top: 2px;
+      margin-right: 2px;
+    }
+
+    .vizLink {
+      @include font-header-xxs;
+      // Override color: inherit in _header.scss.
+      color: $primary-light !important;
+    }
+  }
+}
+
+.body {
+  .stats {
+    display: flex;
+    margin-bottom: 30px;
+
+    .column {
+      flex: 1 1 0;
+      min-width: 0;
+      margin-right: 10px;
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    .metric {
+      display: flex;
+      align-items: center;
+      height: 30px;
+      padding: 0 10px;
+
+      &:first-child {
+        background-color: $off-white;
+      }
+
+      .label,
+      .value {
+        flex: 1 1 0;
+        min-width: 0;
+      }
+
+      .label {
+        @include font-header-xxs;
+      }
+
+      .value {
+        @include font-body-xxs;
+      }
+
+      .ncbiLink {
+        color: $primary-light !important;
```

Comment:
Why do we need important here?

Reply:
To override `a { color: inherit !important; }` in `_header.scss`. Will add a comment.